airflow/providers/databricks/hooks/databricks_base.py

```diff
@@ -113,7 +113,7 @@ def my_after_func(retry_state):
         else:
             self.retry_args = dict(
                 stop=stop_after_attempt(self.retry_limit),
-                wait=wait_exponential(min=self.retry_delay, max=(2 ** retry_limit)),
+                wait=wait_exponential(min=self.retry_delay, max=(2**retry_limit)),
```

Comment:
Hmm that’s an interesting change of style.

Reply:
In https://github.com/psf/black/issues/538 (where the change was made), it mentions that [PEP 8](https://peps.python.org/pep-0008/#other-recommendations) advises to only use whitespace to help make operator precedence more clear (`**` has a high precedence), and to not use it otherwise.